cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -590,9 +578,11 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
         }
     });
 
-    StaffLockEditor = AbstractEditor.extend({
-        templateName: 'staff-lock-editor',
-        className: 'edit-staff-lock',
+    AbstractVisibilityEditor = AbstractEditor.extend({
```

Comment:
All the logic in `AbstractVisibilityEditor` is shared by both `StaffLockEditor` (the `staff_only` checkbox with 2 states used by Section and Unit) and `ContentVisibilityEditor` (the new radio group with 3 states used by Subsection).

Just a bit of DRY-ing